packages/common/src/adapters/collection.ts

```diff
@@ -112,3 +113,25 @@ export const userCollectionMetadataFromSDK = (
 
   return newCollection
 }
+
+export const accountCollectionFromSDK = (
+  input: full.AccountCollection
+): AccountCollection | undefined => {
+  const playlistId = decodeHashId(input.id)
+  const ownerId = decodeHashId(input.user.id)
```

Comment:
nit: can  rename to `userId` since it's in the `user`

Reply:
Fair point, we mix the term `ownerId` in a few places so I think this was just carryover from that.